core/primitives/src/runtime/parameter_table.rs

```diff
@@ -1,23 +1,57 @@
 use super::config::{AccountCreationConfig, RuntimeConfig};
+use near_primitives_core::account::id::ParseAccountError;
 use near_primitives_core::config::{ExtCostsConfig, VMConfig};
 use near_primitives_core::parameter::{FeeParameter, Parameter};
 use near_primitives_core::runtime::fees::{RuntimeFeesConfig, StorageUsageConfig};
+use near_primitives_core::types::AccountId;
 use num_rational::Rational;
-use serde::de::DeserializeOwned;
-use std::any::Any;
 use std::collections::BTreeMap;
 
+/// Represents values supported by parameter config.
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq)]
+#[serde(untagged)]
+pub(crate) enum ParameterValue {
+    Null,
+    U64(u64),
+    Rational { numerator: isize, denominator: isize },
+    String(String),
+}
+
+impl ParameterValue {
+    fn get_u64(&self) -> Option<u64> {
+        match self {
+            ParameterValue::U64(v) => Some(*v),
+            _ => None,
+        }
+    }
+
+    fn get_rational(&self) -> Option<Rational> {
+        match self {
+            ParameterValue::Rational { numerator, denominator } => {
+                Some(Rational::new(*numerator, *denominator))
+            }
+            _ => None,
+        }
+    }
+
+    fn get_string(&self) -> Option<String> {
```

Comment:
Would it make sense to make it `fn get_string(&self) -> Option<&str>`?
(or even  `fn as_string(&self) -> Option<&str>`)

Usually it's better to let the caller decide if a clone is necessary.

Reply:
Definitely, goes well with the change above, done.